Helm-388: Rename Helm in the Helm docs (#653)
OpenNMS/grafana-plugin: docs/modules/getting_started/pages/basic_walkthrough.adoc

= Create a Dashboard

> This section demonstrates how to create a simple dashboard in the {page-component-title}.

Reviewer: ```suggestion
This section demonstrates how to create a simple dashboard using the {page-component-title}.
```

> You must have the following pre-requisites to create a dashboard:

Reviewer: ```suggestion
You must have the following set up to create a dashboard:
```

* An instance of Grafana with the plugin installed.
* An instance of OpenNMS that is reachable from your Grafana instance.
* One or more nodes being monitored by OpenNMS.

If you already use the OpenNMS Data Source plugin with Grafana, see xref:migrating_from_opennms_datasource.adoc[] for information on how to switch to the {page-component-title}.

== Set up the plugin

First, you must enable the plugin in Grafana and <<custom-actions, create custom actions>> to help with alarm diagnosis:

. Navigate to the home page of your Grafana instance.
. In the left navigation menu, select menu:Configuration[Plugins]:
+
image::gf-plugins.png["Grafana menu displaying navigation options. The Plugins option is highlighted.", 250]

. Select the *OpenNMS Plugin for Grafana*.
** If the plugin is not listed, try restarting your Grafana server.

> ** If the issue persists, make sure the plugin is in the Grafana plugins directory (default: `var/lib/grafana/plugins`).

Reviewer: ```suggestion
** If the issue persists, make sure the plugin's files are in the Grafana plugins directory (default: `var/lib/grafana/plugins`).
```

> Before you create a dashboard, you must set up the datasource (either xref:datasources:performance_datasource.adoc[performance], xref:datasources:flow_datasource.adoc[flow], or xref:datasources:entity_datasource.adoc[entity]).

Reviewer: ```suggestion
Before you create a dashboard, you must set up the datasource (xref:datasources:performance_datasource.adoc[performance], xref:datasources:flow_datasource.adoc[flow], or xref:datasources:entity_datasource.adoc[entity]).
```

> For this walk-through, we will configure the entity datasource.

Reviewer: ```suggestion
For this walk-through, we will configure an entity datasource.
```
